pallets/ocex/src/tests.rs

```diff
@@ -651,16 +634,36 @@ fn test_update_trading_pair_bad_origin() {
 	});
 }
 
+#[test]
+fn test_register_trading_pair_volume_too_low() {
+	new_test_ext().execute_with(|| {
+		assert_noop!(
+			OCEX::register_trading_pair(
+				Origin::root(),
+				AssetId::polkadex,
+				AssetId::asset(1),
+				1_u128.into(),
+				100_u128.into(),
+				1_u128.into(),
+				100_u128.into(),
+				100_u128.into(),
+				10_u128.into(),
+			),
+			Error::<Test>::TradingPairConfigUnderflow
+		);
+	});
+}
+
```

Comment:
Please add tests where you are testing edges (e.g. `10001`, `999`). I can see that you are fixed other tests with `10001` but for this one should be defined dedicated test because in other cases requirements could change in advance and this case will be uncovered...

Reply:
And these tests should fail or pass after `min_order_price * min_order_qty` operation so please hold this in mind..

Reply:
Added test case test_register_trading_pair_volume_too_low for testing failing edge case,and  test_open_trading_pair already has the passing edge case